src/io/flutter/sdk/FlutterSettingsConfigurable.java

```diff
@@ -108,27 +83,23 @@ public String getId() {
 
   @Nullable
   @Override
-  public Runnable enableSearch(String option) {
-    //TODO(pq): enable search
```

Comment:
'Flutter' word is successfully searched for automatically. Do you think you'll want to look for anything else? Ready to return TODO back.